examples/grpc-client/openssl.conf

```diff
@@ -0,0 +1,81 @@
+# OpenSSL configuration file.
```

Comment:
Where's this from? Unless you wrote it yourself, please add a comment about its origin.